eng/pipelines/typespec-apiview.yml

```diff
@@ -53,6 +53,6 @@ jobs:
         -RepoName $(Build.Repository.Name) `
         -PullRequestNumber $(System.PullRequest.PullRequestNumber)`
         -Language 'TypeSpec' `
-        -CommitSha $(Build.SourceVersion)
+        -CommitSha $(System.PullRequest.SourceCommitId)
```

Comment:
Why this switch? Do we expect that this pipeline will only work if triggered from a pull request?

Reply:
Yes, this should only be run from a pull request. I think I should also update the pipeline trigger.